pkg/types/config.go

```diff
@@ -23,6 +23,8 @@ import (
 	"strings"
 )
 
+const DefaultProtocol = "http"
```

Comment:
I notice this is exported (capitalized). Does it need to be?

Reply:
Yes, using it from go-mod-bootstrap rather than repeat the constant locally.